### ISSUE TITLE
Missing images in Understanding 1.4.11

### ISSUE THREAD

**Issue by filippo-zorzi** [w3c/wcag#4016]
There are 2 images missing in the[Understanding SC 1.4.11 - Non-text Contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast):

1. "Default focus style", in the "Passing User Interface Component Examples" section
![Missing image on "Default focus style" example](https://github.com/user-attachments/assets/cd46356c-228a-4163-9a09-812119214643)

2. "Colored underline is the only indicator of a link", in the "Failing User Interface Component Examples" section
![Missing image on "Colored underline is the only indicator of a link" example](https://github.com/user-attachments/assets/ef2dff65-703e-4f9d-9753-6285ee9c77d8)

Both images are in the folder /21/img/ in the repo
Both relative path seems fine
Both absolute path seems fine

Seems like the live server doesn't have the images

**Comment by patrickhlauke** [comment]
@kfranqueiro could this be some build-related oddity? the images do show up correctly when looking at WCAG 2.1 https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast but not WCAG 2.2

**Comment by filippo-zorzi** [comment]
In the local build the images are displayed correctly

**Comment by iadawn** [comment]
Thanks folks... don't worry @patrickhlauke you did fix it, I just didn't commit the files into CVS. Missed them 🙄

**Comment by patrickhlauke** [comment]
ah, thanks for clarifying @iadawn
